Sources/UInt128.swift

```diff
@@ -323,8 +284,8 @@ public struct UInt128 {
 // MARK: - UnsignedIntegerType
 extension UInt128: UnsignedInteger {
     public init(_ value: UIntMax) {
-        self.init()
-        self.value.lowerBits = value
+        hi = 0
```

Comment:
Change this to self.value.upperBits and the below to self.value.lowerBits and I do agree that it makes more sense. Though, I do think I should just bite the bullet and make init(upperBits:lowerBits:) as a designated initializer.